### PR TITLE
Add logging

### PR DESCRIPTION
**Pull request by VictoriaRogo**
Trello: https://trello.com/c/PEPmHBcZ

Added log method into BaseTest class
Configured logging before and after a test

Questions:
1. Order of execution of methods decorated by @BeforeMethod and @AfterMethod depends on the method location in the code not intuitively understandable.
2. Any suggestions related to the content of the start and end messeges. 
3. Color highlighting. Currently implemented with ANSI codes and can be not cross-platform.

### CODE REVIEW

File: src/test/java/runner/BaseTest.java [hunk 1]
```diff
@@ -29,7 +35,8 @@
             } finally {
                 con.disconnect();
             }
-        } catch (IOException ignore) {}
+        } catch (IOException ignore) {
+        }
```
Comment (SergeiDemyanenko) [inline src/test/java/runner/BaseTest.java:39]: не стоит делать эти изменения

File: src/test/java/runner/BaseTest.java [hunk 2]
```diff
@@ -5,21 +5,27 @@
...
+import java.util.logging.Logger;
 
 public abstract class BaseTest {
 
     public static final String HUB_URL = "http://localhost:4444/wd/hub";
+    private static final String ANSI_RESET = "\u001B[0m";
+    private static final String ANSI_YELLOW = "\u001B[33m";
 
     private static boolean remoteWebDriver = false;
+
```
Comment (SergeiDemyanenko) [inline src/test/java/runner/BaseTest.java:28]: Этого не должно тут быть

File: src/test/java/runner/BaseTest.java [hunk 3]
```diff
@@ -64,4 +84,19 @@ protected void setDownAll() {
     protected WebDriver getDriver() {
         return driver;
     }
+
+    protected void log(Level level, String msg) {
+        Logger logger = Logger.getLogger(this.getClass().getName());
+        logger.log(level, msg);
+    }
+
+    private String getClassMethodName(Method method) {
```
Comment (SergeiDemyanenko) [inline src/test/java/runner/BaseTest.java:93]: Используй этот вариант - method.toGenericString()

**Review by SergeiDemyanenko — CHANGES_REQUESTED** [review]

File: src/test/java/runner/BaseTest.java (hunk 1 above)
Comment (VictoriaRogo) [reply src/test/java/runner/BaseTest.java]: removed this change

File: src/test/java/runner/BaseTest.java (hunk 2 above)
Comment (VictoriaRogo) [reply src/test/java/runner/BaseTest.java:26]: removed

File: src/test/java/runner/BaseTest.java (hunk 3 above)
Comment (VictoriaRogo) [reply src/test/java/runner/BaseTest.java]: used this.getClass(), method.getName()

**Review by SergeiDemyanenko — APPROVED** [review]

**Pull request merged**